### ISSUE TITLE
Error when updating plugin

### ISSUE THREAD

**Issue by BWBama85** [afragen/github-updater#925]
I am getting this error when trying to update a plugin:

```
PHP Fatal error:  Uncaught TypeError: in_array(): Argument #2 ($haystack) must be of type array, stdClass given in /home/nginx/domains/bamapolitics.com/public/wp-content/plugins/github-updater/src/GitHub_Updater/Branch.php:88
Stack trace:
#0 /home/nginx/domains/bamapolitics.com/public/wp-content/plugins/github-updater/src/GitHub_Updater/Branch.php(88): in_array()
#1 /home/nginx/domains/bamapolitics.com/public/wp-content/plugins/github-updater/src/GitHub_Updater/Base.php(436): Fragen\GitHub_Updater\Branch->set_branch_on_switch()
#2 /home/nginx/domains/bamapolitics.com/public/wp-includes/class-wp-hook.php(287): Fragen\GitHub_Updater\Base->upgrader_source_selection()
#3 /home/nginx/domains/bamapolitics.com/public/wp-includes/plugin.php(212): WP_Hook->apply_filters()
#4 /home/nginx/domains/bamapolitics.com/public/wp-admin/includes/class-wp-upgrader.php(527): apply_filters()
#5 /home/nginx/domains/bamapolitics.com/public/wp-admin/includes/class-wp-upgrader.php(797): WP_Upgrader->install_package()
#6 /home/nginx/domains/bamapolitics.com/public/wp-admin/includes/class-plugin-upgrader.php(345): WP_Upgrader->run()
#7 /home/nginx/domains/bamapolitics.com/public/wp-admin/includes/ajax-actions.php(4452): Plugin_Upgrader->bulk_upgrade()
#8 /home/nginx/domains/bamapolitics.com/public/wp-includes/class-wp-hook.php(287): wp_ajax_update_plugin()
#9 /home/nginx/domains/bamapolitics.com/public/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters()
#10 /home/nginx/domains/bamapolitics.com/public/wp-includes/plugin.php(484): WP_Hook->do_action()
#11 /home/nginx/domains/bamapolitics.com/public/wp-admin/admin-ajax.php(184): do_action()
#12 {main}
  thrown in /home/nginx/domains/bamapolitics.com/public/wp-content/plugins/github-updater/src/GitHub_Updater/Branch.php on line 88
```

**Comment by afragen** [comment]
Does this happen for a specific plugin?

What version of GitHub Updater are you using?

**Comment by afragen** [comment]
You must be using PHP8. Fix coming.

**Comment by afragen** [comment]
This should be fixed in `develop`.

**Comment by afragen** [comment]
Released in 9.9.7
